.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "msedge",
+            "request": "launch",
+            "name": "Launch Edge against localhost",
+            "url": "http://localhost:8082",
+            "webRoot": "${workspaceFolder}/frontend/src/*"
+        }
+    ]
+}
```

Comment:
I don't think we want this file in git. It's accounted for the in the .gitignore in backend and frontend but I think we will need a top level .gitignore as well. Just add the entry for this to start (you can copy from existing files).

Reply:
Fixed.